### ISSUE TITLE
BUG: Export whole table feature doesn't include the primary key

### ISSUE THREAD

**Issue by enricodias** [beekeeper-studio/beekeeper-studio#1214]
**Describe the bug**
When exporting a whole table, the primary key (auto increment) is not included in the export. In addition, each row receives its own `insert into`, which is unnecessary.

**To Reproduce**
Try to export a whole table with a primary key, using the SQL export format.

**Expected behavior**
The whole table should e exported. If you import the resulted sql again, you should get the same table with the same data.

**Comment by rathboma** [comment]
Hey! Can you give me some more information to help debug this? (I can't replicate locally):

- Database type and version
- Some example SQL to create a table and insert data into it to demo the problem

Thanks!

**Comment by enricodias** [comment]
As I was creating a reproducible example I noticed that the export do contains the primary keys, but the values are on alphabetical order and not in the same order as they appear in the table. When I opened this issue I didn't notice that the primary key on my export was way at the end and just I assumed it was missing from the export. Sorry for that 😅

The export still creates one insert query for each row, thought. This could be optimized.
